src/compactor.rs

```diff
@@ -106,7 +106,7 @@ impl CompactorOrchestrator {
     ) -> Result<Self, SlateDBError> {
         let options = Arc::new(options);
         let stored_manifest =
-            tokio_handle.block_on(StoredManifest::load(manifest_store.clone()))?;
+            tokio_handle.block_on(StoredManifest::try_load(manifest_store.clone()))?;
```

Comment:
ditto: can't this just use `load`? It seems like `MissingLatestManifest` is a more appropriate error than `InvalidDBState` anyway